### ISSUE TITLE
Screen reader is not announcing the 'Equations' in solution information while navigating in scan mode.

### ISSUE THREAD

**Issue by msftedad** [mathjax/MathJax#3386]
**Test Environment:**
Window 11 24H2 26100.2605
Edge Browser Version 137.0.3296.93 
Tools used: NVDA, Narrator, JAWS
Test URL: https://codepen.io/trcko/pen/YzpevPm

**User Impact:**
When navigating in scan mode, the screen reader's silence on variables and special characters can significantly affect users who rely on it for information. This lack of auditory feedback can make it difficult for them to understand the content and context of the page, leading to a frustrating and inaccessible experience. Ensuring that all elements are announced is crucial for an inclusive user experience.

**Observation:** 
Issue is observed with all screen readers i.e., Narrator, NVDA and JAWS.
While navigating in browse mode using nvda screen reader is remaining silent when focus is on the variables and special characters in the equation.

**Repro Steps:**

1. Open the URL - https://codepen.io/trcko/pen/YzpevPm on edge browser.
2. Turn on Narrator using ‘Ctrl + Win + Enter’ keys.
3. Press 'Caps lock + spacebar’ key to navigate in scan mode. 
4. Use arrow keys to navigate over the mathematical expression. 
5.  Observe the screen reader announcement and verify whether Screen reader is announcing the 'Equations' in solution information while navigating in scan mode.

**Actual Result:**
While navigating in scan mode screen reader is remaining silent i.e., not announcing any information when focus is on the variables and special characters in the page. For example, when navigating over expressions , the screen reader fails to announce the variable 'x', ‘y’. 

**Refer Attachment:** 
1.While navigating in scan mode screen reader is remaining silent i.e., not announcing any information when focus is on the variables and special characters in the page..png
2.In scan mode, when users navigate over mathematical expressions, the screen reader remains silent.mp4

https://github.com/user-attachments/assets/8da15dda-07a2-4ecc-a470-dd20b744ee55
<img width="960" alt="Image" src="https://github.com/user-attachments/assets/230266ed-f61b-493c-9604-56920d76ba24" />

**Expected result:**
While navigating in scan mode screen reader should be announcing the variables and special characters information in the equation when focus is on the content in the page. 
The screen reader should announce each component of the equation, including variables like 'x,' providing users with accurate mathematical expressions. 
Mathematical symbols and operations should be pronounced correctly to ensure comprehension.

**Comment by msftedad** [comment]
Hi @dpvc,  updated MathJax to version 4.1, the same problem still same with MathJax version 4.1. Just to clarify, the whole equation can be read correctly, but when in scan mode and user tries to do it character by character it doesn't work for some characters. I have tried using MathJax speech-rule-engine and toSpeech() function, which works, but that only generated string for reading the whole equation, which we already have. I have also tried setting enableSpeech option to true, but that did not resolve our problem. Could you please reactivate the bug?

**Comment by dpvc** [comment]
> when in scan mode and user tries to do it character by character it doesn't work for some characters.

I'm not sure what that means.  Can you be more explicit about what is going wrong?

I am using a VM for running Windows, and I can't seem to get Narrator to run, but I do have NVDA and JAWS, and these seem to do what I expect them to do.

For example, if I use the up and down arrows, NVDA will read the math properly as a whole (as you say).  If I use the right and left arrows, it will read the text string of the math one character at a time.  Not particularly helpful, but it is what is expected.

If the NVDA cursor outline is on a math expression (which it has just read), it indicates that it is clickable, and if you press ENTER to perform the "click", you enter the MathJax expression explorer.  At that point, you are in focus mode, and MathJax handles the arrow keys.  Down will move you to a lower level of nesting (up will take you back to higher levels), and right and left will move you among the terms at the same level, reading each as it goes.  Pressing "h" gets you a help dialog that explains how all of this works.  Pressing ESCAPE will move you back out of the expression (and back to browse mode) where NVDA now controls the keys and you can use down to move to the next text in the page.  

All of that is working for me as expected for me.

**Comment by msftedad** [comment]
Hi @dpvc,
Issue is fixed. Thanks

**Comment by dpvc** [comment]
@msftedad, I'm glad you got it worked out.
